src/App.js

```diff
@@ -1,15 +1,16 @@
 import React, { Component } from "react";
 import "./App.css";
-import Box from "./components/Box.js";
 import Header from "./components/Header.js";
 import Footer from "./components/Footer.js";
+import Addbox from "./components/Addbox.js";
 
 class App extends Component {
   render() {
     return (
       <>
         <Header />
-        <Box />
+
```

Comment:
That's cool how you returned the header and footer as their own components. I just returned a h1 tag in the app component itself.